lib/rubocop/cop/correctors/punctuation_corrector.rb

```diff
@@ -5,6 +5,10 @@ module Cop
     # This auto-corrects punctuation
     class PunctuationCorrector
       class << self
+        include RangeHelp
```

Comment:
Added to be able to use a range to check its surrounding bits of code. And `range_between` requires access to the `processed_source`, so this had to be passed to the `Corrector`.